cmd/podman/images_prune.go

```diff
@@ -47,11 +47,11 @@ func pruneImagesCmd(c *cliconfig.PruneImagesValues) error {
 		fmt.Printf(`
 WARNING! This will remove all dangling images.
 Are you sure you want to continue? [y/N] `)
-		ans, err := reader.ReadString('\n')
+		answer, err := reader.ReadString('\n')
 		if err != nil {
 			return errors.Wrapf(err, "error reading input")
 		}
-		if strings.ToLower(ans)[0] != 'y' {
+		if strings.ToLower(answer)[0] != 'y' {
```

Comment:
This right here is an example of a change that I don't think should keep a build from completing.  'answer' is better for sure, but not a stopper in my book.

Reply:
I think this is better.